js/constants/constants.js

```diff
@@ -109,6 +109,17 @@ module.exports = {
     UNMUTE: "Unmute",
     FULLSCREEN: "Fullscreen",
     EXIT_FULLSCREEN: "Exit Fullscreen",
+    SEEK_SLIDER: "Seek slider",
+    VOLUME_SLIDER: "Volume slider",
+    VOLUME_PERCENT: "{volume}% volume",
+    TIME_DISPLAY: "{currentTime} of {totalTime}",
+    TIME_DISPLAY_LIVE: "Live video",
+    TIME_DISPLAY_DVR: "{currentTime} of {totalTime} live video"
```

Comment:
Can we store these macros (currentTime, totalTime, volume) as constants? If we do, we should update the macro replacement functions to use these constants as well.

Reply:
It's a bit complicated to reference other values from the same file since it's defined as an object literal. I'll leave this for another PR since I still have other stuff I need to fix